stac_fastapi/types/stac_fastapi/types/core.py

```diff
@@ -253,9 +254,11 @@ class LandingPageMixin(abc.ABC):
     """Create a STAC landing page (GET /)."""
 
     stac_version: str = attr.ib(default=STAC_VERSION)
-    landing_page_id: str = attr.ib(default="stac-fastapi")
-    title: str = attr.ib(default="stac-fastapi")
-    description: str = attr.ib(default="stac-fastapi")
+    landing_page_id: str = attr.ib(default=os.getenv("STAC_FASTAPI_ID", "0.1"))
+    title: str = attr.ib(default=os.getenv("STAC_FASTAPI_TITLE", "stac-fastapi"))
+    description: str = attr.ib(
+        default=os.getenv("STAC_FASTAPI_DESCRIPTION", "stac-fastapi")
+    )
```

Comment:
To me it feels weird that we have to set this here again but it's not your fault 😓